helmfile.d/helmfile-100.autoconfig.yaml

```diff
@@ -0,0 +1,23 @@
+bases:
+  - snippets/defaults.yaml
+---
+bases:
+  - snippets/env.gotmpl
+---
+bases:
+  - snippets/derived.gotmpl
+---
+{{ readFile "snippets/templates.gotmpl" }}
+{{- $v := .Values }}
+{{- $c := $v.charts }}
+
+releases:
+  - name: job-autoconfig
+    {{ if and (eq $v.otomi.hasExternalIDP false) (eq $c.gitea.enabled true) }}
```

Comment:
You can make is simplier 
```
installed: {{ if and (eq $v.otomi.hasExternalIDP false) (eq $c.gitea.enabled true) }}
```

Reply:
Done :)

installed: {{ and (eq $v.otomi.hasExternalIDP false) (eq $c.gitea.enabled true) }}

Reply:
You are right and you made it perfect! Thanks